src/game/mod.rs

```diff
@@ -24,6 +26,10 @@ impl GameId {
         let id: u128 = rng.gen();
         GameId(id)
     }
+    // Added for API to create a GameId object to input to the GameManager
+    pub fn from(id: u128) -> Self {
```

Comment:
Is `GameId::from(id)` really better than `GameId(id)`?

Reply:
Will put it in the game module